openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -257,11 +257,8 @@ def _handle_successful_authentication_and_login(user, request):
 
     try:
         django_login(request, user)
-        if request.POST.get('remember') == 'true':
-            request.session.set_expiry(604800)
-            log.debug("Setting user session to never expire")
-        else:
-            request.session.set_expiry(0)
+        request.session.set_expiry(604800 * 4)
```

Comment:
@nasthagiri Should we use `SESSION_COOKIE_AGE` setting to set the expiry or is this fine?